### PR TITLE
update crowdin.yml, CHANGELOG.md, fr.json, it.json, nl.json and ro.js…

### PR DESCRIPTION
**Pull request by camilavcoutinho**
#### What is the purpose of this pull request?
update french, dutch, romanian and italian translations

#### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: messages/fr.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "omnishipping.deliveryTitle": "Expédition",
   "omnishipping.loading": "Chargement en cours…",
-  "omnishipping.awaitingData": "En attente de plus d'informations",
+  "omnishipping.awatingData": "En attente de plus d'informations",
```
Comment (lucasecdb) [inline messages/fr.json:4]: why this change?

File: messages/fr.json [hunk 2]
```diff
@@ -52,7 +52,7 @@
   "omnishipping.deliveryWindows": "Du {startDate} au {endDate}",
   "omnishipping.deliveryWindowsSummary": "{date}, entre {startDate} et {endDate}",
   "omnishipping.deliveryWindowsSummaryDate": "{date}, entre {startDate} et {endDate}",
-  "omnishipping.deliveryWindowsSummaryTime": "entre {startDate} et {endDate}",
+  "omnishipping.deliveryWindowsSummarytime": "entre {startDate} et {endDate}",
```
Comment (lucasecdb) [inline messages/fr.json:55]: ditto

File: messages/fr.json [hunk 3]
```diff
@@ -62,16 +62,16 @@
   "omnishipping.change": "Voir tous les sites de retrait en magasin disponibles",
   "omnishipping.searchLocationMap": "Rechercher une rue, un code postal, un quartier ou une ville",
   "omnishipping.productsInPoint": "Produits disponibles dans ce lieu de collecte",
-  "omnishipping.additionalInfo": "Autres informations",
+  "omnishipping.aditionalInfo": "Autres informations",
```
Comment (lucasecdb) [inline messages/fr.json:65]: ditto

File: messages/it.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "omnishipping.deliveryTitle": "Spedizione",
   "omnishipping.loading": "Caricamento in corso…",
-  "omnishipping.awaitingData": "In attesa di maggiori informazioni",
+  "omnishipping.awatingData": "In attesa di maggiori informazioni",
```
Comment (lucasecdb) [inline messages/it.json:4]: ditto

**Review by lucasecdb — CHANGES_REQUESTED** [review]
there are a lot of changes in the messages keys, and I checked the `messages/en.json` file and they're correct there.. is there something out of sync in the crowdin integration that is exposing the old messages keys?

File: messages/fr.json (hunk 1 above)
Comment (dvm1984) [reply messages/fr.json:4]: this is very strange. Maybe source strings in en.json has the typo?

Comment (lucasecdb) [reply messages/fr.json:4]: they don't have the typo, I checked

Comment (dvm1984) [reply messages/fr.json:4]: must revert changes to keys then.

**Review by dvm1984 — APPROVED** [review]

**Review by lucasecdb — APPROVED** [review]

**Pull request merged**